app/assets/stylesheets/utilities/_overrides.scss

```diff
@@ -28,17 +28,28 @@
   }
 }
 
-.govuk-tag {
-  &.govuk-tag--warning {
-    background-color: govuk-colour("yellow", $legacy: "yellow");
-    color: govuk-colour("black", $legacy: "black");
+.broken-link-summary {
```

Comment:
As discussed, these styles should be moved into `objects/_broken-link-status.scss` as they're not overwrites anymore, but new BEM classes and we should prefix them with `.app-` (e.g. `.app-broken-link-status`, `.app-broken-link-status--fail`, `.app-broken-link-status--pass`) to make it clear where the styles come from.

I'm happy to keep them as partials for now.

Reply:
Done 👍